src/topology/continuous_function/basic.lean

```diff
@@ -224,6 +224,8 @@ def restrict (f : C(α, β)) : C(s, β) := ⟨f ∘ coe⟩
 
 @[simp] lemma coe_restrict (f : C(α, β)) : ⇑(f.restrict s) = f ∘ coe := rfl
 
+@[simp] lemma restrict_apply (f : C(α, β)) (s : set α) (x : s) : (f.restrict s) x = f x.val := rfl
```

Comment:
```suggestion
@[simp] lemma restrict_apply (f : C(α, β)) (s : set α) (x : s) : f.restrict s x = f x.val := rfl
```

Reply:
@YaelDillies, I edited your suggestion to include a fix for @sgouezel's comment